PendingReleaseNotes

```diff
@@ -65,3 +65,11 @@ Relevant tracker: https://tracker.ceph.com/issues/55521
   For more details, see:
 
   https://docs.ceph.com/en/latest/mgr/telemetry/
+* Cephadm may now be configured to carry out CephFS MDS upgrades without
+reducing ``max_mds`` to 1. Previously, Cephadm would reduce ``max_mds`` to 1 to
+avoid having two active MDS modifying on-disk structures with new versions,
+communicating cross-version-incompatible messages, or other potential
+incompatibilities. This could be disruptive for large-scale CephFS deployments
+because the cluster cannot easily reduce active MDS daemons to 1.
+Relevant tracker: https://tracker.ceph.com/issues/55715
```

Comment:
Add note that a staggered upgrade of the mons/mgrs may be necessary to take advantage of the feature. Link to that cephadm doc.